lib/anyway/loaders/yaml.rb

```diff
@@ -10,17 +10,40 @@ module Anyway
   module Loaders
     class YAML < Base
       def call(config_path:, **_options)
-        base_config = trace!(:yml, path: relative_config_path(config_path).to_s) { load_base_yml(config_path) }
+        rel_config_path = relative_config_path(config_path).to_s
+        config = trace!(:yml, path: rel_config_path) { load_base_yml(config_path) }
+        if environmental?(config)
+          return trace!(:yml, path: rel_config_path) { config_with_env(config, config_path) }
+        end
 
-        return base_config unless use_local?
+        mix_local!(config, config_path) if use_local?
```

Comment:
What is the goal of this method extraction?

Reply:
For removing duplication. We can mix local config in env configuration too.

Reply:
> We can mix local config in env configuration too.

Yeah, but why moving env configuration into its own method then?

Loading from local config is always the last step; in the original version it was clear; now we have it in two places, our code no longer reflects the actual flow, it's not sequential.